assets/javascript/app.js

```diff
@@ -0,0 +1,214 @@
+// alert("this is a test!")
+
+// for the record, I researched and made up all these questions!! Wine is a passion, loads of fun!
+// Questions array containing 11 objects (made up of question, choices, and answer)
+
+var questions = [
+  {
+    question:
+      "Which type of American grape is equivalent to the Italian Primitivo grape?",
+    choices: [
+      "Cabernet Sauvignon",
+      "Merlot",
+      "Zinfandel",
+      "Pinot Noir",
+      "Petite Syrah"
+    ],
+    answer: "Zinfandel"
+  },
+  {
+    question: "Which of the following red varietals are NOT from Tuscany?",
+    choices: [
+      "Chianti",
+      "Primitivo",
+      "Sangiovese",
+      "Canaiolo Nero",
+      "Brunello di Montalcino"
+    ],
+    answer: "Primitivo"
+  },
+  {
+    question:
+      "Which of the following wood species is NOT used to make barrels for aging wine?",
+    choices: [
+      "American Oak",
+      "French Oak",
+      "European Beachwood",
+      "Hungarian Oak",
+      "California Redwood"
+    ],
+    answer: "European Beachwood"
+  },
+  {
+    question: "Which appelation is NOT located in France?",
+    choices: [
+      "Loire Valley",
+      "Cotes du Rhone",
+      "Bordeaux",
+      "Rioja",
+      "Burgundy",
+      "Sancerre"
+    ],
+    answer: "Rioja"
+  },
+  {
+    question:
+      "Which legendary French winemaker put Napa Valley 'on-the-map' as a world-class wine producer?",
+    choices: [
+      "Robert Mondavi",
+      "Rodney Strong",
+      "Charles Krug",
+      "Georges de Latour",
+      "André Tchelistcheff"
+    ],
+    answer: "Georges de Latour"
+  },
+  {
+    question:
+      "Which American appelation is best known for producing a dry and 'dusty' red grape?",
+    choices: [
+      "Calistoga",
+      "Los Carneros",
+      "Rutherford",
+      "Stags Leap",
+      "Sonoma Valley"
+    ],
+    answer: "Rutherford"
+  },
+  {
+    question: "Which grape is looked down upon as being 'inferior'?",
+    choices: ["Merlot", "Sauvignon Blanc", "Chardonnay", "Cabernet Sauvignon"],
+    answer: "Chardonnay"
+  },
+  {
+    question: "Which description is generally NOT used to describe wine?",
+    choices: [
+      "dry",
+      "fruit-forward",
+      "tanic",
+      "bold",
+      "peppery",
+      "savory",
+      "earthy",
+      "barnyardy"
+    ],
+    answer: "savory"
+  },
+  {
+    question:
+      "Which country is the largest producer of the Sauvignon Blanc grape?",
+    choices: ["France", "Italy", "Australia", "United States", "South Africa"],
+    answer: "France"
+  },
```

Comment:
You may want to keep the formating consistent so that it's easier for you to navigate in the future.